lib/UI.rb

```diff
@@ -0,0 +1,37 @@
+class UserInterface 
```

Comment:
It is better if we just make the User interface as a module than a class. Game, Player and Board are ok to be classes since they satisfy owning data and behaviour and they can be instantiated as many times as needed. But UserInterface isn't an object that holds any data or behavior meaningful to the logic of the game. So it is better as a module than a class.